### ISSUE TITLE
SVG Rechartering 2021

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/strategy#272]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:](https://w3c.github.io/charter-drafts/2021/svg.html)

What kind of charter is this?

 - [X] Existing WG recharter 
 - [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FGraphics%2FSVG%2Fsvg-2019.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2021%2Fsvg.html), and any issue discussion:

Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).

Communities suggested for outreach: 
The existing SVG WG community and the [SVG CG](https://svg-cg.org). 
Accessibility folks. 

Known or potential areas of concern: 
It's unclear how much scope we should give to develop new features or just focus exclusively on maintenance. 

There is a possibility to adopt new work, but there has been little movement in the SVG space in the last few years.  

It's unclear at the moment what we should do with the "SVG Native" deliverable. As it's made no progress in the last 2 years, and it's was an unpublished draft, I'm of the opinion it should be incubated in the SVG CG or WICG.   

Where would charter proponents like to see issues raised?
This strategy funnel issue

Anything else we should think about as we review? 
No

**Comment by himorin** [comment]
it seems:
- charter history seems lacking the latest charter until 2021/March.
- patent policy points 2017 edition but not the newest one (link points the newest one)
- link to process document in 9.1 charter history points 2017 edition

**Comment by samuelweiler** [comment]
@marcoscaceres I kindly suggest starting with the template and stuffing things back into it.  It's a pain.  We need something less painful.  But it's the best process we have for now.  Otherwise, your charter reviewers will need to pick apart every line and say "this doesn't match the agreed-to blah", and that's terribly inefficient and frustrating for the reviewers.  Which isn't to say you can't diverge from the template (for cause), or that the template doesn't need fixing.  Please file copious issues against the template.  :-)

Please tag me again when the boilerplate is cleaned up.

**Comment by marcoscaceres** [comment]
Thanks @himorin... yeah, I copy/pasted an old one, hence all the out of date stuff. And thanks @samuelweiler! I'll do as you suggested (retrofit things into the newer template). 

Related, I filed https://github.com/w3c/respec/issues/3570 a few days ago so we can just get ReSpec to do all the boilerplate. Hopefully it will make this a bit more fun... hoping that we can just create these in markdown, and get ReSpec to spit them out as HTML. If anyone wants to help me with that, it be greatly appreciated. 

Be back soon with an updated draft of this charter!

**Comment by marcoscaceres** [comment]
I've updated to use the newer template. Thanks again @himorin and @samuelweiler for your guidance!

**Comment by samuelweiler** [comment]
Thank you @marcoscaceres.

Substantively, it looks good to me.

Before you send this to W3M, add an end date, decide if you want goal dates on the docs, and clean up the todo tag in section 2 - you probably want the normal text, not the snapshot text?

**Comment by michael-n-cooper** [comment]
No comments from APA; over to @brewerj to complete accessibility horizontal review.

**Comment by himorin** [comment]
No comment/request from i18n.

**Comment by marcoscaceres** [comment]
Soliciting feedback from [www-svg](https://lists.w3.org/Archives/Public/www-svg/2021Jun/0005.html).

**Comment by plehegar** [comment]
Should we move SVG-AAM to the ARIA WG? Looks like the ARIA folks are interested. See also https://github.com/w3c/strategy/issues/282

**Comment by michael-n-cooper** [comment]
ARIA WG is also rechartering https://github.com/w3c/strategy/issues/282 and suggests SVG-AAM be moved to the ARIA WG, with coordination via liaison statements to SVG.

**Comment by marcoscaceres** [comment]
FWIW, that seems reasonable to me.

**Comment by marcoscaceres** [comment]
Sent a preemptive PR about moving SVG-AAM to the ARIA WG: https://github.com/w3c/charter-drafts/pull/377

**Comment by svgeesus** [comment]
AC review was 2021-11-30 to 2022-01-10.

**Comment by marcoscaceres** [comment]
Any guidance on what should happen (or what I should do) next? 🙏

**Comment by plehegar** [comment]
Announced
